apps/backend/src/libs/types/service.type.ts

```diff
@@ -5,7 +5,7 @@ type Service<T = unknown> = {
 	findAll(): Promise<{
 		items: T[];
 	}>;
-	update(id: number, payload: unknown): Promise<T>;
+	update(id: number, payload: Partial<T>): Promise<T>;
```

Comment:
@fshabanov Should we use Partial<> or unknown here?

Reply:
I think Partial is better